src/quo2/components/onboarding/small_option_card/component_spec.cljs

```diff
@@ -40,15 +39,4 @@
       (h/test "`:icon` variant"
         (h/render (testing-small-option-card :icon component-props))
         (h/fire-event :press (h/get-by-label-text :small-option-card))
-        (-> on-press-fn js/expect .toHaveBeenCalled))))
-
-  (h/describe "Image rendered"
```

Comment:
@J-Son89 im not sure how we could fix that, so i removed this test

Reply:
probably we could move it to quo-preview ?

Reply:
oh yeah, we could either move it to preview or just mock the data needed. I don't think it cares whether it's actually an image or not for the test.

```
let (image { :uri "mock-uri" ...
```

Reply:
if you want to move forward with this pr, you can also `skip` the test for now. I could handle it in a follow up if it helps.
https://jestjs.io/docs/api#testskipname-fn